src/helpers/misc.ts

```diff
@@ -468,6 +468,11 @@ export function removeFalsyAttributes(obj: Object): Object {
   return cleanObject;
 }
 
+/** Returns the sum of an array of number */
+export function sumOfArray(arr: number[]) {
```

Comment:
can be deleted

Reply:
I managed to add this helper twice (once in the data fitlers PR), and twice at the end I never used it :)